### PR TITLE
Update cds and ctdc models

### PR DESCRIPTION
**Pull request by n2iw**

### CODE REVIEW

File: cache/CTDC/1.0.0/ctdc_model_file.yaml [hunk 1]
```diff
@@ -2,7 +2,6 @@
 # Title case names are "reserved" (meaningful to the parser)
 # Lower case names are labels for the entities
 
-Version: v.1.0.0
```
Comment (amattu2) [inline cache/CTDC/1.0.0/ctdc_model_file.yaml:5]: The CTDC Version attribute got removed (probably it isn't in CTDC repo yet).

**Review by amattu2 — CHANGES_REQUESTED** [review]
I just did a (very) quick check of Model Navigator and both models load as expected. I annotated one issue with the CTDC update, the Version prop got removed.

File: cache/CTDC/1.0.0/ctdc_model_file.yaml (hunk 1 above)
Comment (n2iw) [reply cache/CTDC/1.0.0/ctdc_model_file.yaml:5]: Good catch Alec! I'll let Gina know.

**Review by amattu2 — APPROVED** [review]
LGTM!

(Did not re-verify in model navigator)

**Pull request merged**